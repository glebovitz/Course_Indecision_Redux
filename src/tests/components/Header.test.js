import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../components/Header';

test('should render an Header Component correctly', () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
});

test('should render an Header Component correctly', () => {
  const wrapper = shallow(<Header title="Test Title" subtitle="Test Subtitle"/>);
  expect(wrapper).toMatchSnapshot();
});
