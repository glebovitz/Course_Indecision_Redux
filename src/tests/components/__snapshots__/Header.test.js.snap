// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render an Header Component correctly 1`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      some default
    </h1>
  </div>
</div>
`;

exports[`should render an Header Component correctly 2`] = `
<div
  className="header"
>
  <div
    className="container"
  >
    <h1
      className="header__title"
    >
      Test Title
    </h1>
    <h2
      className="header__subtitle"
    >
      Test Subtitle
    </h2>
  </div>
</div>
`;
